docs/topics/sb-features-and-frameworks-summary.adoc

```diff
@@ -46,10 +46,10 @@ a|
 Discovers Service/Endpoint deployed on OpenShift and exposed behind a service or route using the service name matching a DNS entry.
 
 a|
-Yes - using Kubernetes API
+Yes
```

Comment:
I think this reference to the Kubernetes API might actually be correct (not sure), it's the first one (in the Circuit Breaker section) that I can't possibly see being right.

Reply:
Yes, discovery is also done using Kubernetes API so we can keep that one.